index.html

```diff
@@ -84,7 +84,7 @@ <h3>Background and Motivation</h3>
                 <img src="images/text-input-process.jpg" alt="">
                 <figcaption>A sequence diagram illustrating a typical flow for text input.</figcaption>
             </figure>
-            <p>Existing user agents handle the details of this text input process so that the author’s responsibility ends at declaring what elements of the document represent an editable region.  Authors express which regions are editable using input elements, textarea elements, contenteditable elements, or by setting the designMode attribute to true to mark an entire document as editable. </p>
+            <p>Existing user agents handle the details of this text input process so that the author’s responsibility ends at declaring what elements of the document represent an editable region.  Authors express which regions are editable using input elements, textarea elements, contenteditable elements, or by setting the {{Document/designMode}} attribute to "<code>on</code>" to mark an entire document as editable. </p>
```

Comment:
Should this be the [activeDocument](https://html.spec.whatwg.org/multipage/document-sequences.html#active-document) instead? Also, can a web author attach `EditContext` to a stray `Document` that doesn't have [focus](https://html.spec.whatwg.org/multipage/interaction.html#has-focus-steps)?

Reply:
> Should this be the [activeDocument](https://html.spec.whatwg.org/multipage/document-sequences.html#active-document) instead?

Can you clarify? Which part should be activeDocument? The change to `{{Document/designMode}}` just makes `designMode` into a link to the definition in the DOM spec.

> Also, can a web author attach EditContext to a stray Document that doesn't have [focus](https://html.spec.whatwg.org/multipage/interaction.html#has-focus-steps)?

This will work, similarly to how authors can set `contenteditable` on elements in non-windowed documents. It just won't do much because that document won't receive input.

Reply:
> This will work, similarly to how authors can set contenteditable on elements in non-windowed documents

Ah, I guess it doesn't matter then whether the Document that the EditContext is attached to can receive text input focus or not.